asb-ballerina/Module.md

```diff
@@ -18,7 +18,7 @@ applies [SAS authentication](https://docs.microsoft.com/en-us/azure/service-bus-
 supports SAS authentication as well.
 
 This module
-supports [Service Bus SDK 3.5.1 version](https://docs.microsoft.com/en-us/java/api/overview/azure/servicebus/client?view=azure-java-stable&preserve-view=true)
+supports [Service Bus SDK 7.13.1 version](https://docs.microsoft.com/en-us/java/api/overview/azure/servicebus/client?view=azure-java-stable&preserve-view=true)
```

Comment:
Let's update the link to the latest SDK as well.
https://learn.microsoft.com/en-us/java/api/overview/azure/messaging-servicebus-readme?view=azure-java-stable